2021/infoTalk.html

```diff
@@ -51,7 +51,7 @@
             <div class="col-lg-9 col-sm-9 col-xs-12 site-content-left">
                 <div class="category-post-list post-list single-list">
 
-                    <article cass="post-294 post type-post status-publish format-standard has-post-thumbnail hentry category-culturel tag-business tag-news tag-congress tag-entertainment">
+                    <article class="post-294 post type-post status-publish format-standard has-post-thumbnail hentry category-culturel tag-business tag-news tag-congress tag-entertainment">
```

Comment:
This is it right? All the other changes r just formatting

Reply:
Yes, the attribute had a typo, the correct one is `class`. You can check the preview in the screenshot I attached above.